thrift/thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -162,8 +164,9 @@ public ServiceSpecification generate(List<Entry> entries, DocServiceFilter filte
 
     @VisibleForTesting
     @Nullable
-    ServiceInfo newServiceInfo(Class<?> serviceClass, Iterable<EndpointInfo> endpoints,
+    ServiceInfo newServiceInfo(Entry entry,
                                DocServiceFilter filter) {
```

Comment:
style; the two lines can be merged
```suggestion
    ServiceInfo newServiceInfo(Entry entry, DocServiceFilter filter) {
```

Reply:
Get it!